.github/workflows/pr_build_images.yaml

```diff
@@ -14,3 +14,19 @@ jobs:
           docker build --file pulp_ci/Containerfile --tag pulp/pulp-ci:latest .
           docker build --file pulp_fedora31/Containerfile --tag pulp/pulp-fedora31:latest .
           docker build --file pulp_galaxy_ng/Containerfile --tag pulp/pulp-galaxy-ng:latest .
+      - name: Test the images
+        run: |
+          mkdir settings pulp_storage pgsql containers
+          echo "CONTENT_ORIGIN='http://$(hostname):8080'
+                ANSIBLE_API_HOSTNAME='http://$(hostname):8080'
+                ANSIBLE_CONTENT_HOSTNAME='http://$(hostname):8080/pulp/content'
+                TOKEN_AUTH_DISABLED=True" >> settings/settings.py
+          docker run --detach pulp/pulp-fedora31:latest \
+                     --publish 8080:80 \
+                     --volume ./settings:/etc/pulp:Z \
+                     --volume ./pulp_storage:/var/lib/pulp:Z \
+                     --volume ./pgsql:/var/lib/pgsql:Z \
+                     --volume ./containers:/var/lib/containers:Z \
```

Comment:
In my experience you only really need the settings volume. But maybe we want to test the way we document it...